WebDriverAgentLib/Categories/XCUIElement+FBWebDriverAttributes.m

```diff
@@ -161,4 +161,15 @@ - (NSDictionary *)wdRect
   };
 }
 
+- (NSDictionary *)wdRectNoInf
+{
+  CGRect frame = self.wdFrame;
+  return @{
+           @"x": @((frame.origin.x != INFINITY) ?frame.origin.x :0),
```

Comment:
such values are returned by XCTest and I don't think it makes a lot of sense to build such workarounds over them. I'd rather try to find the reason of why it happens. Perhaps, it is some wrong way of reading element attributes in WDA source or a bug on the Apple side?

Reply:
My debugging says it's an Apple bug. When the Links are not visible, at least the ones at the bottom of the Springboard, they show the INFINITE values. I need the springboard tree, so this patch helps.

Reply:
why not to use `isinf`?

Reply:
Nicer with isinf, thanks.